asmcli/lib/util.sh

```diff
@@ -244,6 +244,26 @@ prepare_environment() {
   fi
 }
 
+# Need to prepare differently under multicluster environment
+# validate_cluster and configure_kubectl will be called in validation
+# for each cluster
+prepare_create_mesh_environment() {
```

Comment:
If this is create_mesh specific I think we can leave it in the command file for now.

Reply:
Done